src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -155,7 +155,7 @@ rm -f /etc/resolv.conf > /dev/null 2>&1
 echo "nameserver $DNS_SERVER" > /etc/resolv.conf
 
 apt-get update > /dev/null
-apt-get install curl -y > /dev/null 2>&1
+apt-get install curl openssh-server -y > /dev/null 2>&1
```

Comment:
It's good to have DEBIAN_FRONTEND specified for the non-interactive runs. E.g.:
`DEBIAN_FRONTEND=noninteractive apt-get install curl openssh-server -y > /dev/null 2>&1`
￼